src/packages/cli/src/__tests__/update-message.test.ts

```diff
@@ -0,0 +1,23 @@
+import 'ts-node/register'
+import { printUpdateMessage } from '../utils/printUpdateMessage'
+import { consoleContext, Context } from './__helpers__/context'
+
+const ctx = Context.new().add(consoleContext()).assemble()
+
+it('should display a update message', async () => {
```

Comment:
Great test!

I would also add a test for the latest (the one users are actually using) too 😃